tests/suite/utils/custom_resources_utils.py

```diff
@@ -163,7 +163,7 @@ def patch_gc_from_yaml(custom_objects: CustomObjectsApi, name, yaml_manifest, na
     :return: a dictionary representing the resource
     """
     print(f"Load {yaml_manifest}")
-    return patch_custom_resource_v1alpha1(custom_objects, name, yaml_manifest, namespace, "globalconfigurations")
+    return patch_custom_resource_v1(custom_objects, name, yaml_manifest, namespace, "globalconfigurations")
```

Comment:
Opening a discussion on this one.
From what I see we only ever had `patch_custom_resource_v1alpha1` and never had a generic ``patch_custom_resource` in this file.

So we could keep this change as is or change it to `patch_custom_resource` and also add a `patch_custom_resource_v1alpha1` to have both

Reply:
we can have one function now `def patch_custom_resource()` then below listed ones can use that:
`def patch_ts()`
`def patch_gc_from_yaml()`
`def patch_ts_from_yaml()`

```
def patch_custom_resource_v1alpha1(custom_objects: CustomObjectsApi, name, yaml_manifest, namespace, plural) -> None:
    """
    Patch a custom resource based on yaml manifest
    """
    print(f"Update a Resource: {name}")
    with open(yaml_manifest) as f:
        dep = yaml.safe_load(f)

    try:
        custom_objects.patch_namespaced_custom_object("k8s.nginx.org", "v1", namespace, plural, name, dep)
    except ApiException:
        logging.exception(f"Failed with exception while patching custom resource: {name}")
        raise
```